localstack/services/sqs/sqs_listener.py

```diff
@@ -196,9 +196,9 @@ def get_external_port(headers, request_handler):
     return request_handler.proxy.port
 
 
-def validate_empty_message_batch(data):
+def validate_empty_message_batch(data, req_data):
     data = to_str(data).split('Entries=')
-    if len(data) > 1 and data[1] == '':
+    if len(data) > 1 and data[1] == '' and not req_data.get('Entries'):
```

Comment:
Thanks for updating the PR. I think this should be:
```
     if len(data) > 1 and not req_data.get('Entries'):
        ...
```

Reply:
if not req_data.get('Entries'):

This statement is always true no matter Entries are passed or not.So there is no use to add it is what I feel.What do you say?